serialization/Cargo.toml

```diff
@@ -7,9 +7,13 @@ license = "MIT"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
+serialization-derive = { path = 'derive' }
+
 arraytools = "0.1.5"
 hex-literal = "0.3.4"
 parity-scale-codec = {version = "3.1.2", features = ["derive", "chain-error"]}
+rand = "0.8.5"
```

Comment:
I guess this should be moved to the crypto subsystem randomness source

Reply:
Note that this is here because serialization cannot depend on the crypto crate because the crypto crate depends on serialization. If we want to brute-force the randomness dependency, we should move random out of crypto to its own crate.

Reply:
Gotcha. Makes sense to keep it imported directly then.